attendance/lib/backend/user.dart

```diff
@@ -29,7 +29,7 @@ class User {
   /// rename the user (change native name)
   void rename(String newName) {
     final String trimmed = newName.trim();
-    if (_isValidName(trimmed)) {
+    if (isValidName(trimmed)) {
```

Comment:
@Bahaaib no need to make`isValidName` public, I guess you can just catch the exception and deal with it